ClassDiagram/ClassDiagram.md

```
@@ -19,12 +24,13 @@
 8. [Booking System](#8-booking-system)
 9. [Delivery System](#9-delivery-system)
 10. [Payment System](#10-payment-system)
-11. [Administration & Audit](#11-administration--audit)
-12. [Relationship Map](#12-relationship-map)
-13. [Business Rules](#13-business-rules)
-14. [Delivery Model Matrix](#14-delivery-model-matrix)
-15. [Booking Status State Machine](#15-booking-status-state-machine)
-16. [Registration Compatibility](#16-registration-compatibility)
+11. [Rating System](#11-rating-system)
+12. [Administration & Audit](#12-administration--audit)
+13. [Relationship Map](#13-relationship-map)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)